datasets/bow_processors/reuters_processor.py

```diff
@@ -6,7 +6,7 @@
 class ReutersProcessor(BagOfWordsProcessor):
     NAME = 'Reuters'
     NUM_CLASSES = 90
-    VOCAB_SIZE = 36308
+    VOCAB_SIZE = 36311
```

Comment:
This should be due to a mismatch in the scikit library version in your environment. In any case, rather than hard-coding the vocabulary size, would it be possible to infer it at run time?

Reply:
The vocab size is hard coded for all the datasets. Should I add a method to calculate vocab size in the`BagOfWordsProcessor` class that's extended for each dataset?

Reply:
Yup that would work